pkg/retry/config_test.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package retry
+
+import (
+	"testing"
+
+	"github.com/abcxyz/pkg/testutil"
+)
+
+const (
+	testAppID              = "test-app-id"
```

Comment:
It's usually more common to hardcode these in the tests. Constants in tests would be used when you have a real external resource, like an upstream URL.